NorthstarDLL/squirrel/squirrel.cpp

```
@@ -376,12 +375,15 @@ void __fastcall ScriptCompileErrorHook(HSquirrelVM* sqvm, const char* error, con
 		// kill dedicated server if we hit this
 		if (IsDedicatedServer())
 		{
+			logger->error("Exiting dedicated server, compile error is fatal");
 			// flush the logger before we exit so debug things get saved to log file
 			logger->flush();
 			exit(EXIT_FAILURE);
 		}
 		else
 		{
+			logger->error("Disconnecting, compile error is fatal");
```

possibly remove these? not much point and might hide the actual error in logs